day_6/exercises/burrito.rb

```diff
@@ -1,19 +1,45 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    self.toppings = @toppings.push(toppings)
+    puts "Now your toppings include #{toppings}!"
+  end
+
+  def remove_topping(remove_topping)
+    # you can also add "" to the toppings variable
+    self.toppings -= [@toppings.delete(remove_topping)]
```

Comment:
How could you simplify this statement? Do you need to reassign self.toppings? Same for the add_topping method above